cityofaustin/atd-moped
# Add phase template modal to Timeline tab

## Associated issues

https://github.com/cityofaustin/atd-data-tech/issues/11495

This PR doesnt address phase ordering. The moped_proj_phases table doesn't have a phase_order column whereas the moped_proj_milestones table does. 

~~I merged https://github.com/cityofaustin/atd-moped/pull/1006 into this, those files changed will be visible in here until that's merged to main.~~ its been merged to main

## Testing
**URL to test:** 
https://deploy-preview-1003--atd-moped-main.netlify.app/moped/projects/1686?tab=timeline

you can test via netlify link, since the db changes have been merged to main

**Steps to test:**

1. Navigate to a project's timeline tab. 
2. Click on the ADD PHASE dropdown button 
3. Try adding a phase with "+ NEW PHASE" to add phases like normal
4. Also try the "FROM TEMPLATE" option. A modal should appear. Choose "Arterial Management" as your template
5. Add all of the phases
6. In a different project, try with selecting only some of the phases and add them
7. When you open the modal again, the ones you didnt add should be available to add again. Previously selected phases from the template should not be an option to add. 


---
#### Ship list
